apps/dsp-app/cypress/support/commands.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />

```

Comment:
why do we have interfaces in this file ?

Reply:
those are used in the commands, but I can move it to separate file, how do you prefer?

Reply:
yes please!